tests/functional/mhd_2d_vector_potential_init/case.py

```
@@ -0,0 +1,151 @@
+RECON = os.environ.get("PHARE_RECON", "WENOZ")
+LIMITER = os.environ.get("PHARE_LIMITER", "None")
+
+gamma = 5.0 / 3.0
+
+# --- geometry: dipole singularity OUTSIDE the [0,L]^2 domain (smooth B0 inside) ---
+L = 4.0
+dipole_center = (-1.5, -1.5)
+m_dipole = 1.0
+b1_imf = 0.2  # uniform perturbation field along -y (IMF in B1)
+
+density = lambda x, y: 1.0 + 0.0 * x
```

Ruff (0.15.18)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `density` as a `def`

(E731)